build/macos-x64/install.dependencies.sh

```diff
@@ -5,7 +5,7 @@ export HOMEBREW_NO_AUTO_UPDATE=1
 
 brew list --formula
 
-brew install autoconf automake libtool nasm ossp-uuid pkg-config ragel
+brew install autoconf automake cmake libtool nasm ossp-uuid pkg-config ragel
```

Comment:
Why was this added? This already worked?

Reply:
As I mentioned in the initial comment, I tested this first on a new macos VM and installed homebrew from scratch. In a default install, cmake isn't installed (it's installed by default on the github runners). I'm happy to remove this if we want the scripts to only run on the github runners, but in my experience if cmake is already installed this operation has no effect anyway

Reply:
I would rather only install things that we need to install on the runner. So please remove this.